README.md

```diff
@@ -298,4 +298,16 @@ Requirements:
 
 ```bash
 npm run get-asset-vault-details
+```
+
+#### 5.4 Get asset vault details
```

Comment:
I think you forgot to change the header.

`Get asset vault details` -> `Sync asset attributes`

Reply:
Yup. Changes made